chainset.go

```diff
@@ -66,6 +66,21 @@ func (cs *chainSet) Initialize(ctx context.Context, testName string, cli *client
 	return eg.Wait()
 }
 
+func (cs *chainSet) Configuration(ctx context.Context, testName string, additionalGenesisWallets map[ibc.Chain][]ibc.WalletData) error {
```

Comment:
Can you add comment code for this func. I think it might be difficult for first-time users of our tool to understand